### PR TITLE
allow specifying a timing threshold

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

**Review by emord — APPROVED** [review]
The output of this would be one line each for the setup, test and tear down, correct? So it would only output something if one of those steps was > threshold? (not actionable, just want to make sure I understand it)

**Comment by snopoke** [comment]
@emord that's correct

File: corehq/tests/noseplugins/timing.py [hunk 1]
```diff
@@ -32,6 +32,9 @@ def options(self, parser, env):
                           default=env.get('NOSE_PRETTY_TIMING'),
                           help='Print timing info in a format that is better '
                                'for reviewing in text mode (not CSV).')
+        parser.add_option('--threshold', type=int,
+                          default=env.get('NOSE_TIMING_THRESHOLD'),
+                          help='Only print timing info for events above this threshold.')
```
Comment (millerdev) [inline corehq/tests/noseplugins/timing.py]: Assuming the threshold units are seconds. Might be nice to document that here.

**Review by millerdev — APPROVED** [review]

**Comment by czue** [comment]
does travis print these out for us?

**Comment by snopoke** [comment]
Not at present but I do plan on adding that in.

**Review by benrudolph — APPROVED** [review]

**Pull request merged**